src/restLink.ts

```diff
@@ -908,6 +918,7 @@ export class RestLink extends ApolloLink {
         resolverOptions,
       )
         .then(data => {
+          setContext({ restResponses: requestContext.responses });
```

Comment:
Hmmm, this is problematic if users are merging queries from more than one `RestLink`. Is this something we need to worry about?

Reply:
Do you mean they might have more than one `RestLink` instance in the link/middleware chain? I had not considered that, but I think that could be handled by detecting the presense of `restResponses` and merging the arrays. The logic for using it upstream should all still hold with this change.